spec/models/post_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'rails_helper'
+
+# Checks if all errors come from given attribute , returns false if otherwise
+def TestAttributeOfPost(obj , attribute)
```

Comment:
Tev nevajag papildus metodes rakstit un Ruby pasaulē metode rakstā ar snake_case.

Katrs rails modelis atbild ar metodi .valid? kura izsauc visas validacijas.
Kāpēc tev šis bija vajadzigs?

Reply:
Nomainisu uz snake case , un to metodi izmantoju lai katrā testa nevajadzetu parbaudit to vai nav kādi citi errori besides tos kurus expecto , ja ir tad vins pieliek [ERROR] tagu pirms rezultata.